### DISCUSSION TITLE
About parameter intensity

### DISCUSSION THREAD

**Discussion by machuanfeng** [BindsNET/bindsnet#625] [Q&A] [answered]
Why should the value of parameter intensity be set to 128 instead of 256

**Comment by rafaelblevin821** [comment, marked as answer]
> Why should the value of parameter intensity be set to 128 instead of 256

I think the value of intensity is arbitrary. It is a spike intensity multiplier that amplifies the difference between spike values. You can set it to 256 if you like.

**Reply by machuanfeng** [reply]
OK，thank you very much!
